### DISCUSSION TITLE
Help needed!

### DISCUSSION THREAD

**Discussion by ZeroAsher** [Leather128/LeathersFunkinEngine#229] [Q&A] [answered]
We need help putting a new tutorial into our mod, but we’ve tried multiple times but we keep getting the message “[file_contents,assets/data/tutorial!/tutorial!-hard.json]”
I don’t know what these means and I really need help as soon as possible since I was going to show this to a few people tomorrow. Can someone put what I need into my mod for me? I would greatly appreciate it. Or at least tell me what I need to do.

**Comment by Leather128** [comment, marked as answer]
I think you may have named tutorial as `tutorial!` instead of `tutorial` it probably just doesn't show up in freeplay or anything because `!` might just not show up in freeplay text.

TLDR: Check the `_append/data/freeplaySonglist.txt` and check Tutorial's name there as I think it's incorrect.
